Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -59,13 +59,6 @@ - (void)commontInit
 {
     OWSAssert(!self.avatarView);
 
-    const CGFloat kMinVMargin = 5;
-
-    [self setTranslatesAutoresizingMaskIntoConstraints:NO];
-    self.layoutMargins = UIEdgeInsetsMake(0, self.cellHMargin, 0, self.cellHMargin);
-    self.contentView.layoutMargins = UIEdgeInsetsZero;
-    self.contentView.preservesSuperviewLayoutMargins = YES;
-
     self.backgroundColor = [UIColor whiteColor];
```

Comment:
The intention is that as we adopt `UITableViewAutomaticDimension` / auto-sizing of cells throughout our views, to converge upon "cutting with the grain" of how iOS auto layout and `UITableViewCells` are supposed to work.

* Don't `preservesSuperviewLayoutMargins`.
* Don't meddle with cell or `cell.contentView` default layout margins.
* Pin to margins, not edges.